client/src/components/MainApp/Sidebar/SidebarMobile/SidebarMobile.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
Instead of making a new component for smaller devices. Could you make a custom hook in `client/src/utils/useWindowSize.js` which calculates the window size every time it's resized? Here's a link for [reference](https://usehooks.com/useWindowSize/). This way you can leverage conditional rendering to show the playlists and Spotify logo on the sidebar (desktop version) and remove them on small devices. 

That's something I would be interested in rather than making separate components for desktop and mobile devices

Reply:
Also do add this repo as **upstream** and pull the latest commits. 

**If you're stuck here's a guide to help you [setting up upstream](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork)**